### ISSUE TITLE
Web project templates that ship in VS 2017 do not reference MSTest V2 nuget packages

### ISSUE THREAD

**Issue by shyamnamboodiripad** [Microsoft/testfx#167]
## Description
LUT does not work out of the box for web projects with unit tests created using VS 2017. Considering that web projects are used a lot in enterprise and LUT is key selling points for VS Enterprise, LUT should work out of the box for new web projects with unit tests. But it doesn't currently.

## Steps to reproduce
1. Create C# (or VB) web application - "ASP.NET Web Application (.NET Framework)". You may need to install the appropriate willow workload to get Web projects.
2. Check the "Add unit tests check box".
![image](https://cloud.githubusercontent.com/assets/10579684/25413531/eb0c460c-29df-11e7-9028-580d831f227d.png)
3. Turn on LUT (Test->Live Unit Testing->Start).

## Expected behavior
LUT is started

## Actual behavior
```
[17:48:58.018 Error] No test adapters are referenced by this solution. If you have a test project, add a NuGet reference to a test adapter corresponding to the test framework used in order to run the tests. If you already have the required NuGet reference in your test project, performing a NuGet restore may resolve the issue.
[17:48:58.021 Info] Start failed.
[17:48:58.029 Info] Live Unit Testing stopped.
```
Looks like project template for the created unit test project is still adding a reference to the old 'MS.VS.QualityTools.UnitTestFramework.dll' instead of 'MS.VS.TestPlatform.TestFramework' nuget packages. 

## Environment
VS Build: d15PreRel.26423.1

**Comment by shyamnamboodiripad** [comment]
Related to #132

**Comment by shyamnamboodiripad** [comment]
Not sure if we have more templates that suffer from this issue. Will you be reviewing all existing unit test templates that ship in VS 2017 and move all of them over to the v2 nuget packages?

**Comment by AbhitejJohn** [comment]
Thanks for filing this. AFAIK there aren't any others. We might want to check if F# has a UT template too.

**Comment by jayaranigarg** [comment]
Filed a bug on Web Templates team [here](https://devdiv.visualstudio.com/DevDiv/_workitems?id=430394&_a=edit) to track this.

**Comment by ManishJayaswal** [comment]
@AbhitejJohn we should probably keep this issue open to track the fix in the internal bug db and close it it once it is fixed.

**Comment by AbhitejJohn** [comment]
@ManishJayaswal : Thought the internal bug could be used. I can leave this open if it makes things more clear.

**Comment by ManishJayaswal** [comment]
@AbhitejJohn thanks! This helps folks from community who are tracking this issue.

**Comment by jayaranigarg** [comment]
This has been fixed in 15.3 latest Preview.
